flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -289,4 +298,27 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
         result("")
         return
     }
+
+    private func fetchNativeAppStart(result: @escaping FlutterResult) {
+        if let appStartMeasurement = PrivateSentrySDKOnly.appStartMeasurement {
+
+            let appStartTime = appStartMeasurement.appStartTimestamp.timeIntervalSince1970 * 1000
+            let isColdStart = appStartMeasurement.type == .cold
+
+            let item: [String: Any] = [
+                "appStartTime": appStartTime,
+                "isColdStart": isColdStart,
+            ]
+
+            result(item)
+        } else {
+            result(
+                FlutterError(
+                    code: "1",
+                    message: "App start won't be sent due to missing appStartMeasurement",
+                    details: nil
+                )
```

Comment:
Is this going to error the App? Maybe we just log out instead of throwing an error.

Reply:
This will return the error to the caller on the Flutter side. We then log it in the `NativeWrapper` class and return `null`.

Reply:
Usually, errors are more expensive than returning "nothing", what do you think?

Reply:
I don't know the impact, it's just a hunch.
The difference is, we return a `FlutterError` when the SDK is unable to do its work, e.g. `captureEnvelope` or `initNativeSdk`.
The `appStart` feature is just a measurement, so the SDK is still running and working, hence just logging would be fine, it's rather more semantical than a bug.

Reply:
Can we log in the integrations? I'm not sure we can get hold of the logger instance, not seeing it beeing used?

Reply:
You can pass options to the integrations and the options have the logger. Quite a few integrations access the options.

Reply:
The Swift plugin logs out using the `print` method since it's just for debugging only anyway.
You can also use this approach, or as @ueman said.

Reply:
@ueman Sorry, i confused integrations with the native plugins here. I meant the latter and am now logging directly in the plugin if there was an issue.

<img width="771" alt="Bildschirmfoto 2022-03-09 um 13 49 31" src="https://user-images.githubusercontent.com/3984453/157446930-6ed87d5b-3f32-4cd0-80be-c80435464ddf.png">